src/shared/modules/favorites/favoritesDuck.js

```diff
@@ -60,7 +65,7 @@ export default function reducer (state = initialState, action) {
       return mergeFavorites(initialState, updatedFavorites)
     case LOAD_FAVORITES:
     case UPDATE_FAVORITES:
-      return mergeFavorites(initialState, action.favorites)
+      return mergeFavorites(action.favorites, state)
```

Comment:
I had to change this to ensure favorites got updated. Still not sure how it worked before, or if this is right